how-to-guides/data-versioning/notebooks/Organize_and_share_dataset_versions.ipynb

```diff
@@ -33,7 +33,7 @@
     "\n",
     "[See this example in Neptune](https://app.neptune.ai/o/common/org/data-versioning/metadata?path=datasets%2Ftrain_sampled&attribute=latest)\n",
     "\n",
-    "![image](https://neptune.ai/wp-content/uploads/Screenshot-from-2021-12-23-11-46-23.png)"
+    "![image](https://user-images.githubusercontent.com/41324509/156204557-0878b92e-4ef3-4978-869d-e841d47d9f18.png)"
```

Comment:
all images should be on our wordpress, not github

Reply:
the WP link is broken..also, I don't think I've access to the Neptune WP

Reply:
@Patrycja-J can you help us here?

We need to add [this image](https://user-images.githubusercontent.com/41324509/156204557-0878b92e-4ef3-4978-869d-e841d47d9f18.png) to our WP and pass link here?

Reply:
@Patrycja-J if ok, simply pass link to WP here as a comment